### DISCUSSION TITLE
Can we make a SPA 'installable' ?

### DISCUSSION THREAD

**Discussion by realtebo** [quasarframework/quasar#13110] [General - Components / Directives / etc] [answered]
I'm not sure if it's a PWA-related question or not, sorry.

We made a SPA for a customer. He asked to add 'installable' feature, to have an icon in the homepage of the smartphone. 

Is it possible?
Where is this documented?

**Comment by MilosPaunovic** [comment, marked as answer]
You can install only PWA apps, but you can add that mode pretty easy in exiting Quasar SPA. Check official guide [https://quasar.dev/quasar-cli-webpack/developing-pwa/introduction](https://quasar.dev/quasar-cli-webpack/developing-pwa/introduction)

**Reply by realtebo** [reply]
Ok. It's a starting point.
there is nothing about installation but I'll google

**Reply by yusufkandemir** [reply]
@realtebo that page explains the mode, adding the PWA mode is explained in the following related pages. See: https://quasar.dev/quasar-cli-webpack/developing-pwa/preparation, it's simply `quasar mode add pwa` then use `quasar dev -m pwa` to run the dev server, and `quasar build -m pwa` to build it.

The installable feature will be enabled automatically as long as you are running in secure contexts(localhost, HTTPS, etc.).

**Reply by realtebo** [reply]
The last sentence is what I was missing. I didn't know the installable feature will be enable automatically.
